.github/workflows/test.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       - name: Style
         run: |
           pycodestyle
+          flake8
```

Comment:
`pycodestyle` keep or kill?

Reply:
Does it do anything useful that flake8 doesn't?

Reply:
https://pypi.org/project/flake8/

> Flake8 is a wrapper around these tools:
>
>-    PyFlakes
>-    pycodestyle
>-   Ned Batchelder’s McCabe script
> 
> Flake8 runs all the tools by launching the single flake8 command. It displays the warnings in a per-file, merged 
> output.
> 
> It also adds a few features: 
> ...

If that's true, there's no reason to keep pycodestyle as well.